src/pydrex/minerals.py

```diff
@@ -381,16 +382,24 @@ def update_orientations(
         for minerals undergoing plastic deformation. Return the updated deformation
         gradient measuring the corresponding macroscopic deformation.
 
-        Args:
-        - `config` (dict) — PyDRex configuration dictionary
-        - `deformation_gradient` (array) — 3x3 deformation gradient tensor
-        - `get_velocity_gradient` (function) — callable with signature f(t, x) that
-          returns a 3x3 velocity gradient matrix at time t and position x (3D vector)
-        - `pathline` (tuple) — tuple consisting of:
-            1. the time at which to start the CPO integration (t_start)
-            2. the time at which to stop the CPO integration (t_end)
-            3. callable with signature f(t) that returns the position of the mineral at
-                time t ∈ [t_start, t_end]
+        .. note:: For multi-phase simulations, the return value should only be collected
+            for the last phase, i.e. the last call to `update_orientations`. This value
+            should be passed on to subsequent calls to `update_orientations` as the
+            `deformation_gradient` argument.
```

Comment:
Just putting a quick comment here that we perhaps need to think of a way to change this.

Reply:
Maybe a new `Pathline` object that can store `.x1`, `.x2`[, `.x3`], `.t`, `.strain`, and `.regime`, or anything else that is "macroscopic" history and doesn't fit well as a `Mineral` attribute.

Reply:
Could make `get_pathline` return that object instead of the tuple.

Reply:
I will get this merged for now. However, it seems more straightforward to me that a user only needs to call `update_orientations` once. In that sense, this method could be modified to handle multiple phases in one call, or it could be renamed to `update_orientations_single_phase` and the new `update_orientations` would wrap around it and handle the eventual multiple phases. In any case, these approaches would guarantee a single return tuple, which would prevent confusion.